backend/src/feed/connector.rs

```diff
@@ -83,20 +83,21 @@ impl FeedConnector {
                     hash => self.chain_hash = hash,
                 }
 
-                self.aggregator.send(Subscribe {
-                    chain: payload.into(),
-                    feed: ctx.address(),
-                })
-                .into_actor(self)
-                .then(|res, actor, _| {
-                    match res {
-                        Ok(true) => (),
-                        // Chain not found, reset hash
-                        _ => actor.chain_hash = 0,
-                    }
-                    async {}.into_actor(actor)
-                })
-                .wait(ctx);
+                self.aggregator
+                    .send(Subscribe {
+                        chain: payload.into(),
+                        feed: ctx.address(),
+                    })
+                    .into_actor(self)
+                    .then(|res, actor, _| {
+                        match res {
+                            Ok(true) => (),
+                            // Chain not found, reset hash
+                            _ => actor.chain_hash = 0,
```

Comment:
dq: is `chain_hash == 0 removed later/treated as an error?

Reply:
I've renamed it to `chain_label_hash`. It's been a while since I touched it (it's all just formatting), but in essence the hash is there to prevent the feed (browser client) from subscribing to the same channel twice (nothing bad would happen necessary, but it does prevent spamming the backend and getting the list of all nodes over and over again).

Setting the hash to `0` is making the check for hash equality always* fail, so that you can subscribe to a chain after first attempt failed (there were no nodes yet after backend reboot f.e.).

* as long as no chain name produces FNV hash of 0, which is astronomically unlikely.

Reply:
> * as long as no chain name produces FNV hash of 0, which is astronomically unlikely.

When TelemetryCoin (TECO)?